### PR TITLE
Add Start and Stop methods to Manager

### PR DESCRIPTION
**Pull request by mwielgus**
- remove `Housekeep` and `HouskeepModel` from the interface as they are impl specific.

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -49,18 +42,27 @@ type Manager interface {
...
+	model         model.Model
+	sinkManager   sinks.ExternalSinkManager
+	sinkUris      Uris
+	lastSync      time.Time
+	resolution    time.Duration
+	decoder       sink_api.Decoder
+	mainStopChan  chan struct{}
+	sinkStopChan  chan<- struct{}
+	modelStopChan chan struct{}
+	modelDuration time.Duration
```
Comment (piosz) [inline manager/manager.go]: how about moving "modelDuration" somewhere above to somehow group all fields?

**Comment by piosz** [comment]
LGTM just a nit

File: manager/manager.go (hunk 1 above)
Comment (mwielgus) [inline manager/manager.go]: done

**Comment by vishh** [comment]
LGTM. Thanks for the cleanup @mwielgus

**Pull request merged**